kroxylicious-systemtests/pom.xml

```diff
@@ -22,6 +22,10 @@
         The intention of this module is to test Kroxylicious against a kafka instance deployed in a kubernetes cluster/minikube
         using Strimzi, to simulate a real scenario for end-to-end testing treating Kroxylicious as a black box.
     </description>
+
+    <properties>
+        <kafka.version>3.6.1</kafka.version>
```

Comment:
I'm not comfortable having a different version here. Having a pass in the system tests does not mean that it will work fine for kafka 3.7.0, so from testing point of view it does not make any sense to give that result.